### PR TITLE
feat: add operator --version flag

### LINKED ISSUE #422: feat(operator): add --version flag to display binary version
# Description

- We need to implement a `--version` flag in our operator binary to display the version of the binary being run.

### PR DESCRIPTION
**Pull request by NicolasRampoldi**
# Description

- This PR adds the `--version` flag to the operator binary.

# To Test

- Run `make build_operator`.
- Run `./operator/build/aligned-operator --version`.
- It should display: `Aligned Layer Node Operator version v0.1.6`.
- To change the value of the version, change `OPERATOR_VERSION` at the top of the makefile. This should be done before each release of the operator to the desired binary version.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,8 @@ OS := $(shell uname -s)
 
 CONFIG_FILE?=config-files/config.yaml
```
Comment (MauroToscano) [inline Makefile:6]:
```suggestion
Match the release "0.1.6"
```

**Review by MauroToscano — COMMENTED** [review]
Add the command to the operator README

File: Makefile (hunk 1 above)
Comment (NicolasRampoldi) [reply Makefile:6]: Changed version in [d8b86ec](https://github.com/yetanotherco/aligned_layer/pull/423/commits/d8b86ec5e8afd35967a186b33f781dff77b21d7a).

Comment (NicolasRampoldi) [reply Makefile:6]: Added to readme in [1ab8eed](https://github.com/yetanotherco/aligned_layer/pull/423/commits/1ab8eed756779d80707289513ef36f4bec77b48b).

**Comment by entropidelic** [comment]
lgtm, we should think of a way to check that we don't forget to update `OPERATOR_VERSION` variable in the Makefile

**Review by entropidelic — APPROVED** [review]

**Review by uri-99 — APPROVED** [review]

**Review by MauroToscano — APPROVED** [review]

**Pull request merged**